api/services/gameService.js

```diff
@@ -73,8 +73,15 @@ module.exports = {
 
     if (game.players) {
       if (game.players.length < 2) {
-        throw new Error({ message: 'Cannot populate game without two players' });
-      }
+        /*no game was found*/
+        if(game.players.length == 0){
+          return { status: 400, message: 'Bad Request: Cannot find a game' };
+        }
+        else{
+          throw new Error({ message: 'Cannot populate game without two players' });
+        }
+      } 
+
```

Comment:
This is the right idea, but I think we should instead handle this in the consuming endpoint itself. The error from this issue is specifically triggered by a request to spectate using the /game/spectate endpoint which is managed by the `api/controllers/game/spectate.js` file. I think what we want to do is basically update that file so that its initial call to `gameService.populateGame()` and essentially the entire method is wrapped in the existing `try/catch`, that way this error is caught and transformed to a badRequest response, which that endpoint already does.